xbmc/interfaces/legacy/ModuleXbmc.cpp

```diff
@@ -197,34 +197,41 @@ namespace XBMCAddon
           return lang;
         }
       case CLangCodeExpander::ISO_639_1:
+      {
+        if (region)
         {
-          std::string langCode;
-          g_LangCodeExpander.ConvertToISO6391(lang, langCode);
-          if (region)
-          {
-            std::string region = g_langInfo.GetRegionLocale();
-            std::string region2Code;
-            g_LangCodeExpander.ConvertToISO6391(region, region2Code);
-            region2Code = "-" + region2Code;
-            return (langCode += region2Code);
-          }
-          return langCode;
+          return (StringUtils::Format("{}-{}", g_langInfo.GetLanguageISO6391(),
+                                      g_langInfo.GetCurrentRegionISO31661Alpha2()));
```

Comment:
unnecessary brackets
```suggestion
          return StringUtils::Format("{}-{}", g_langInfo.GetLanguageISO6391(),
                                     g_langInfo.GetCurrentRegionISO31661Alpha2());
```

Reply:
Done.